Updated GitHub desktop fork instructions (#36172)
github/docs: data/reusables/desktop/forking-a-repo.md

> In {% data variables.product.prodname_desktop %}, if you clone a repository that you don't have write access to, and then attempt to push new commits to the repository, a fork will be created for you.

Author: ```suggestion
In {% data variables.product.prodname_desktop %}, if you clone a repository that you do not have write access to, and then attempt to push a change to the repository, a fork will be created for you.
```
